bindings/dart/rust/Cargo.toml

```diff
@@ -46,3 +46,6 @@ tokio = { version = "1.0", features = ["full"] }
 
 [lib]
 crate-type = ["staticlib", "cdylib"]
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(frb_expand)'] }
```

Comment:
Should this be a bug in upstream?

Reply:
I don't think it's a bug, I just follow the upstream example

https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/frb_example%2Fdart_minimal%2Frust%2FCargo.toml#L12-L13

Reply:
ok, I mean the generator should not generate code in this way. But I agree that it's a good fix.